conda/conda-build/meta.yaml

```diff
@@ -114,8 +115,10 @@ requirements:
     - rust
     - ninja
     - cmake {{ cmake_version }}
-    - {{ compiler('c') }} =11.2
-    - {{ compiler('cxx') }} =11.2
+    - {{ compiler('c') }} =11.2   # [linux]
+    - {{ compiler('cxx') }} =11.2 # [linux]
+    - {{ compiler('c') }}   # [osx]
```

Comment:
Can you align the selectors so `# [osx]` is under `# [linux]`

Reply:
Done.